src/zone-patch/index.js

```diff
@@ -48,61 +48,68 @@ function wrapTestInZone(testBody) {
   if (testBody === undefined) {
     return;
   }
-  return testBody.length === 0
-    ? () => testProxyZone.run(testBody, null)
-    : done => testProxyZone.run(testBody, null, [done]);
+  return function(...args) {
+    return testProxyZone.run(testBody, null, args);
+  };
 }
 
+/**
+ * bind describe method to wrap describe.each function
+ */
 const bindDescribe = originalJestFn =>
-  function() {
-    const eachArguments = arguments;
-    return function(description, specDefinitions, timeout) {
-      arguments[1] = wrapDescribeInZone(specDefinitions);
-      return originalJestFn.apply(this, eachArguments).apply(this, arguments);
+  function(...eachArgs) {
+    return function(...args) {
+      args[1] = wrapDescribeInZone(args[1]);
+      return originalJestFn.apply(this, eachArgs).apply(this, args);
+    };
+  };
+
+/**
+ * bind test method to wrap test.each function
+ */
+const bindTest = originalJestFn =>
+  function(...eachArgs) {
+    return function(...args) {
+      args[1] = wrapTestInZone(args[1]);
+      return originalJestFn.apply(this, eachArgs).apply(this, args);
     };
   };
 
 ['xdescribe', 'fdescribe', 'describe'].forEach(methodName => {
   const originaljestFn = env[methodName];
-  env[methodName] = function(description, specDefinitions, timeout) {
-    arguments[1] = wrapDescribeInZone(specDefinitions);
-    return originaljestFn.apply(this, arguments);
+  env[methodName] = function(...args) {
+    args[1] = wrapDescribeInZone(args[1]);
+    return originaljestFn.apply(this, args);
   };
   env[methodName].each = bindDescribe(originaljestFn.each);
   if (methodName === 'describe') {
     env[methodName].only = env['fdescribe'];
     env[methodName].skip = env['xdescribe'];
-    env[methodName].only.each = bindDescribe(originaljestFn.only.each);
-    env[methodName].skip.each = bindDescribe(originaljestFn.skip.each);
```

Comment:
Are the `only.each` and `skip.each` not required anymore? I tested it and it worked, but just to get an insight from you side.

Reply:
Sure, because here https://github.com/thymikee/jest-preset-angular/pull/340/files/bf1a11ee365a9431318d6e3c960b43745b767677#diff-561fc1ab4bca44fdd10cdb2851f0b291R86-R87

`env[methodName].only = env['fit']` and `env[methodName].skip = env['xit']`, and `fit.each` and `xit.each` has been patched, so we don't need to patch `only.each` and `skip.each` again.

Reply:
Perfect! Thanks!